frontend/src/pages/homePage.tsx

```diff
@@ -102,9 +112,241 @@ const NewDayComponent: React.FC = () => {
   );
```

Comment:
The "CHANGE MOOD" button that's shown on the homepage after the user does the daily check-in, currently doesn't do anything. Can you implement that functionality?